### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chawes13**
- Add status to Fuel
- Add LP Subsection Generator
- Add status to LP HOC
- Add Flight Readiness Review
- Minor cleanup

### CODE REVIEW

**Comment by chawes13** [comment]
@bhennes2 Deprecating `lp-hoc` may be a bit controversial. I'm personally not aware of use cases today that aren't covered by hooks or custom HOCs.

Have you used this library recently? What use cases am I perhaps glossing over?

File: README.md [hunk 1]
```diff
@@ -14,21 +14,23 @@ Project | Language | Visibility | Description
...
-[Fuel](https://github.com/LaunchPadLab/fuel) | Ruby, Rails | Public | Stop developing Rails blogs and start writing your actual blog posts with this dead simple blogging engine for Rails.
+[Flight Readiness Review](https://github.com/LaunchPadLab/flight_readiness_review) | Bash | Private | Utility to set up a macOS laptop for design and development at LaunchPad Lab.
+[Fuel](https://github.com/LaunchPadLab/fuel) (no longer maintained) | Ruby, Rails | Public | Stop developing Rails blogs and start writing your actual blog posts with this dead simple blogging engine for Rails.
 [LP Components](https://github.com/LaunchPadLab/lp-components) | JS, React | Public | A set of reusable React components.
 [LP Form](https://github.com/LaunchPadLab/lp-form) | JS, Redux | Public | Extensions for the reduxForm HOC.
-[LP HOC](https://github.com/LaunchPadLab/lp-hoc) | JS, React | Public | A set of higher order components (HOC) for use in React apps.
+[LP HOC](https://github.com/LaunchPadLab/lp-hoc) (deprecated) | JS, React | Public | A set of higher order components (HOC) for use in React apps.
```
Comment (bhennes2) [inline README.md:21]: Do we want to note that `hooks` is the replacement?

File: README.md [hunk 2]
```diff
@@ -14,21 +14,23 @@ Project | Language | Visibility | Description
 --- | --- | --- | ---
 [Client Template](https://github.com/LaunchPadLab/client-template) | JS, React, Redux | Private | Template app for React/Redux SPAs.
 [Decanter](https://github.com/LaunchPadLab/decanter) | Ruby, Rails | Public | Gem that makes it easy to transform incoming data before it hits the controller action or model.
-[Fuel](https://github.com/LaunchPadLab/fuel) | Ruby, Rails | Public | Stop developing Rails blogs and start writing your actual blog posts with this dead simple blogging engine for Rails.
+[Flight Readiness Review](https://github.com/LaunchPadLab/flight_readiness_review) | Bash | Private | Utility to set up a macOS laptop for design and development at LaunchPad Lab.
```
Comment (bhennes2) [inline README.md:17]: 🧑‍🚀

File: README.md (hunk 1 above)
Comment (chawes13) [reply README.md:21]: We could make the language stronger / change the sequencing in the LP HOC library? https://github.com/LaunchPadLab/lp-hoc#a-note-about-hooks

Comment (bhennes2) [reply README.md:21]: I think what we have works for now!

**Review by bhennes2 — APPROVED** [review]
LGTM!

**Pull request merged**